.swiftformat

```diff
@@ -1,5 +1,5 @@
 # File options
---swiftversion 5.9
+--swiftversion 5.10
```

Comment:
❓ If we update this to `5.10` should we also change the used Xcode version from `15.2` to `15.4`?
As far as I saw it is on `15.2` ATM.